.travis.yml

```diff
@@ -50,20 +50,25 @@ matrix:
   - env: EMBER_TRY_SCENARIO=ember-canary
 
 before_install:
-- "export DISPLAY=:99.0"
-- "sh -e /etc/init.d/xvfb start"
 - npm config set spin false
 - npm install -g coveralls pr-bumper
+- $(npm root -g)/pr-bumper/.travis/maybe-check-scope.sh
 
 install:
 - $(npm root -g)/pr-bumper/.travis/maybe-install.sh
 
+before_script:
+- "export DISPLAY=:99.0"
+- "sh -e /etc/init.d/xvfb start"
+- sleep 3 # give xvfb some time to start
+
```

Comment:
Copying boilerplate from other repos in [ciena-blueplanet](https://github.com/ciena-blueplanet), which is where `pr-bumper` lives.